### DISCUSSION TITLE
Seasonal Operator Stats

### DISCUSSION THREAD

**Discussion by PixelNinja132** [danielwerg/r6api.js#63] [Q&A] [answered]
I'm guessing if this was possible, it would have been added but just checking, is it possible to add stats for a player's operators for the current season?

**Comment by danielwerg** [comment, marked as answer]
Yes and no. With new API we could get operators stats for specific season if we know date of start and end of a season, problem is you can only get stats for past 120 days (average season length is ~93 days). There also a few other issues, see #42.
